utils/getCategoriesOption.js

```diff
@@ -0,0 +1,10 @@
+const getCategoriesOptions = (categories) => {
+  if (categories) {
+    const filtredItems = categories.map((item) => (item === 'null' ? null : item))
+
+    return filtredItems
+  } else {
+    return
+  }
+}
+module.exports = getCategoriesOptions
```

Comment:
try to use this, I refactored it a little bit:

const getCategoriesOptions = (categories) => 
  categories ? categories.map((item) => (item === 'null' ? null : item)) : []

Reply:
Unfortunately it didn't work because if we receive just array without id, query don`t give us data